apps/back/src/Expense.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class Expense {
+    private string $name;
+    private \DateTime $date;
+    private string $description;
+    private int $amount;
+    private array $participants;
+    private User $payer;
+
+    public function __construct(string $name, DateTime $date, string $description, int $amount, array $participants, User $payer)
+    {
+        $this->name = $name;
+        $this->date = $date;
+        $this->description = $description;
+        $this->amount = $amount;
+        $this->participants = $participants;
+        $this->payer = $payer;
+    }
+
+    public function setName(string $name){
```

Comment:
```suggestion
    public function setName(string $name)
    {
```

same for all methods